source/administration/changelog.md

```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Fixed Mattermost not loading on Firefox if the `media.peerconnection.enabled` setting in Firefox is set to false.
+ - Fixed login screen sometimes flashing before Mattermost server loads.
+ - Fixed an issue where leaving channel in one tab redirected other channels on other tabs to Town Square as well.
+ - Fixed error code/message and panic when creating post with bad props.
+ - Fixed an issue where bot messages from the Zoom plugin ignored the Zoom API URL field for on-prem Zoom servers.
+ - Updated initial scrolling on post list.
+ - Fixed a problem with the last version of webrtc-adapter.
+ - Fixed case sensitive mention highlighting.
+ - Disabled remove button after the first click to prevent multiple request.
+ - Disabled pull-to-refresh feature on Android (Chrome) to prevent unwanted page refresh.
+ - Fixed an issue with missing validation when an invalid credential was used.
+ - Fixed an issue where clicking `Save` in `Rename Channel` modal without changes did nothing.
+ - Fixed an issue where pasting files into a channel didn't work.
+ - Fixed an issue where Mattermost crashed when posting a code snippet containing white space.
+ - Fixed an issue where a validation error message did not get cleared when switching between creating a private/public channel.
+ - Fixed emoji picker search should not be case-sensitive.
+ - Fixed timestamp not clickable in desktop mobile view.
+ - Fixed deleting a Team via the API breaks the Web UI.
+ 
+### Compatibility
+
+#### Removed and Deprecated Features
+
+- All API v3 endpoints have been deprecated, and scheduled for removal in Mattermost v5.0.
+- The permanent query parameter of the DELETE `/teams/{team_id}` APIv4 endpoint for permanently deleting a team is scheduled for removal in Mattermost v4.7. // XXX Jason needs to update
+- The `mentionKeys` prop in post type plugins is now removed to fix case sensitive mention highlighting. Plugins can retrieve the `mentionKeys` prop from the store as needed.
+
+#### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under `ServiceSettings` in `config.json`:
+    - Added `"ImageProxyType": ""`, `"ImageProxyOptions": ""`, and `"ImageProxyURL": ""` to ensure posts served to the client will have their markdown modified such that all images are loaded through a proxy when these keys are configured.
+    - Added `"ExperimentalGroupUnreadChannels": true` as `default_on` or `"ExperimentalGroupUnreadChannels": false` as `disabled` settings to show an unread channel section in the webapp sidebar.
+    - Added `"VaryByUser": false`, a user-based rate limiting, to rate limit on token and on userID. // XXXX This is under `RateLimitingSettings`
```

Comment:
Should be under `RateLimitingSettings`, not `ServiceSettings`

Reply:
Fixed